applications/nrf5340_audio/src/modules/display/nrf5340_audio_dk_display.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/display.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/kernel.h>
+#include <zephyr/zbus/zbus.h>
+#include <zephyr/logging/log.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <stdint.h>
+
+#include <lvgl.h>
+#include "tab_menu.h"
+#include "tab_log.h"
+#include "macros_common.h"
+
+#define DISPLAY_THREAD_PRIORITY	  5
+#define DISPLAY_INIT_PRIORITY	  91 /* Must have lower priority than LVGL init */
+#define DISPLAY_THREAD_STACK_SIZE 4096
+
+LOG_MODULE_REGISTER(display_log, CONFIG_DISPLAY_LOG_LEVEL);
+
+struct k_thread display_data;
+k_tid_t display_thread;
+lv_obj_t *tab_log;
+
+ZBUS_CHAN_DECLARE(button_chan);
+ZBUS_CHAN_DECLARE(le_audio_chan);
+
+ZBUS_OBS_DECLARE(le_audio_evt_sub_display);
+
+K_THREAD_STACK_DEFINE(display_thread_STACK, DISPLAY_THREAD_STACK_SIZE);
+
+void nrf5340_audio_dk_display_update_thread(void *arg1, void *arg2, void *arg3)
+{
+
+	int update_counter = 0;
+	int update_counter_menu = 0;
+
+	while (1) {
+		update_counter++;
+		if (update_counter >= 10) {
+			update_counter_menu = tab_menu_update(update_counter_menu);
```

Comment:
The use of `update_counter_menu` is a bit confusing. Would it look better if you remove all of the counter logic from `tab_menu_update`, and just keep the logic in this thread? I.e. `tab_menu_update` will only be called every 5th iteration.

Reply:
The reason it is implemented as such is that i considered adding a config that chooses the refresh rate of the vu bar depending on user preference